### DISCUSSION TITLE
How to restart Wyze camera from Home Assistant automation.

### DISCUSSION THREAD

**Discussion by musabbari** [mrlt8/docker-wyze-bridge#1060] [Q&A] [answered]
How can I have an automation to restart the Wyze camera? My doorbell seems to stop registering alerts after a day or so, and a restart fixes it. I would like to restart it every night automatically from Home Assistant. I can restart the camera via the Docker Wyze Bridge interface, but I am not sure how to call it via an automation script.
Thanks!

**Comment by g13092** [comment]
It's not a complete answer, but I saw over in issues that someone was using rest api calls to do this.  I was thinking the same, if HA could send the call, you'd be in good shape:

https://github.com/mrlt8/docker-wyze-bridge/issues/1037#issuecomment-1872358096

**Comment by elijah286** [comment, marked as answer]
I created rest commands in home assistant to do this, like so:
<img width="669" alt="image" src="https://github.com/mrlt8/docker-wyze-bridge/assets/7592470/28c965d1-47dd-4752-8cd3-15cfe5a5d997">

**Reply by musabbari** [reply]
Thanks a lot. I got it to work with this template.
